xgettext.js

```diff
@@ -164,7 +164,9 @@ XGettext.prototype._discoverMatches = function( parsedInput ) {
 
 	traverse( parsedInput.ast, {
 		CallExpression: function( node ) {
-			var functionName = ( node.callee.property ) ? node.callee.property.name : node.callee.name;
+			// Pull the resultingFunction out of (0, resultingFunction)()
+			var callee = ( node.callee.type === 'SequenceExpression' ) ? _.last( node.callee.expressions ) : node.callee;
```

Comment:
I looked at the target `(0,f)('string')` and reasoned that it's still a call expression, it's just that the function part (callee) is `(something, f)` instead of just `f`, so I made a tiny file with the example on it, put a `debugger;` statement into xgettext where the change is, and had a look at the data that acorn was generating there.

The [sequence expression](https://github.com/estree/estree/blob/master/spec.md#sequenceexpression) is
> A sequence expression, i.e., a comma-separated sequence of expressions.

The expression as a whole is still a CallExpression, a function call, so I'm not trying to match a new pattern, I'm just pseudo-evaluating the comma expression to pull the relevant part out of it.